### DISCUSSION TITLE
Streamlink download error?

### DISCUSSION THREAD

**Discussion by hyunsu1123** [streamlink/streamlink#5400] [Q&A] [answered]
### Description

[cli][info] Found matching plugin showroom for URL https://www.showroom-live.com/r/rio_omori
error: The specified stream(s) 'showroom' could not be found.
       Available streams: 144p (worst), 360p (best)
[cli][info] Found matching plugin showroom for URL https://www.showroom-live.com/r/rio_omori
error: The specified stream(s) 'showroom' could not be found.
       Available streams: 144p (worst), 360p (best)

It's coming out like this, but I can't download it. Can you tell me why it's like this?

**Comment by bastimeyer** [comment]
https://github.com/streamlink/streamlink/issues/5389

**Reply by hyunsu1123** [reply]
What file should I change?

**Reply by bastimeyer** [reply, marked as answer]
None. Install the latest nightly build, build yourself from the master branch, or wait for the next stable release
